recipe/meta.yaml

```diff
@@ -34,7 +34,8 @@ outputs:
         # The below is needed to get convenience symlinks for users to invoke.
         - compilers
         - cmake
-        - coverage >=3.6
+        # Coverage 6.3 is breaking with NFS and multi-process testing.
+        - coverage !=6.3
```

Comment:
I think it's still good to have `>=3.6,`.  That actually came from the Spectractor requirements.